interface/presenter/poke_presenter.go

```diff
@@ -0,0 +1,23 @@
+package presenter
+
+import "github.com/ToteEmmanuel/academy-go-q12021/domain/model"
+
+type pokePresenter struct {
+}
+
+type PokePresenter interface {
+	ResponsePokemon(*model.Pokemon) *model.Pokemon
+	ResponsePokemons([]*model.Pokemon) []*model.Pokemon
+}
+
+func NewPokePresenter() PokePresenter {
+	return &pokePresenter{}
+}
+
+func (pP *pokePresenter) ResponsePokemon(pokething *model.Pokemon) *model.Pokemon {
+	return pokething
+}
+
+func (pP *pokePresenter) ResponsePokemons(pokethings []*model.Pokemon) []*model.Pokemon {
```

Comment:
Also, is this package necessary if you're not doing anything with data anyway in your functions??

Reply:
Removed it as it really added nothing to logic.